shared/oae/bundles/email/default.properties

```diff
@@ -8,4 +11,8 @@ ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_MULTIPLE_ACTORS = New activity is waiting for y
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_MULTIPLE_ACTIVITIES = ${actorLink} has been very active recently
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_SINGLE_ACTIVITY = ${actorLink} has been active recently
 ACTIVITY_EMAIL_SUMMARY_WEEKLY = Here's what happened in the last week
-
```

Comment:
Unfortunately, the email verification means that we can no longer do development without having emails enabled. Is it worth doing a config flag for development? If not, we should update our README to indicate that enabling emails is required.